RoundCheckbox.js

```diff
@@ -27,19 +32,90 @@ export default class RoundCheckbox extends React.PureComponent {
     onValueChange: () => {},
   };
 
+  state = {
+    scaleAndOpacityOfCheckbox: null,
+  };
+
+  componentDidMount() {
+    this.setState({
```

Comment:
I'd prefer `scaleAndOpacityOfCheckbox` to not be in `state` but just as an instance variable initialized in constructor, thank you

Reply:
I can make this change. I originally tried to do that with a `createRef`, but it wasn't playing nice. It's been a while since I've written class based components. I'll take another stab at this

Reply:
Sure enough, even without `createRef`, I was having issues with a single instance using the following code:

```javascript
  scaleAndOpacityOfCheckbox = null;

  componentDidMount() {
    this.scaleAndOpacityOfCheckbox = new Animated.Value(
      this.props.checked ? 1 : 0,
    );
  }
```

Both checkboxes are "true" at the first. I don't know a way around this without using state. The PR is enabled edits from maintainers, so I welcome changes on it - I'm just not sure with how to progress